patches/bootstrap.py

```diff
@@ -27,6 +27,11 @@ def bootstrap_dash_app(url_base_pathname):
         eager_loading=True,
     )
 
+    # Asynchronously install sklearn (it will be unavailable the first couple of seconds after xiplot has loaded)
```

Comment:
Is there a way to enqueue this delayed install directly inside Python instead of the Python->JS->Python roundtrip?

Reply:
`micropip.install` is async and requires a roundtrip to the javascript event loop (and I haven't found a way around that).